src/lammpsio/snapshot.py

```diff
@@ -84,14 +84,20 @@ def from_hoomd_gsd(cls, frame):
             box=box,
             step=frame.configuration.step,
         )
-        snap.position = frame.particles.position
-        snap.velocity = frame.particles.velocity
-        snap.image = frame.particles.image
-        snap.typeid = frame.particles.typeid + 1
-        snap.charge = frame.particles.charge
-        snap.mass = frame.particles.mass
-
-        snap.molecule = frame.particles.body + 1
+        if frame.particles.position is not None:
+            snap.position = frame.particles.position
```

Comment:
Please add whitespace between these to improve readability
```suggestion
            snap.position = frame.particles.position

```